### DISCUSSION TITLE
anyone knows what ucrt.pem is?

### DISCUSSION THREAD

**Discussion by Echeveria911** [smallstep/certificates#1324] [Q&A]
Hey, I have a MacBook Pro and I'm trying to create a **ucrt.pem** file to skip the activation lock

can any one help me with it?

**Comment by maraino** [comment]
According to [this](https://support.apple.com/guide/security/localpolicy-signing-key-creation-management-sec1f90fbad1/web) you need to create a certificate request using the OIK, and send it to the "Basic Attestation Authority" that would be something in Apple infrastructure to create the actual certificate.

**Reply by Echeveria911** [reply]
man thank you for your reply..
I annoyed you on ur email, wishing to get a reply from you :)
thanks!!
